_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
+    the details of its scripts below the surface, away from public view,
+    and is very efficient at storing those scripts.
+
+- **Schnorr predates ECDSA:** we talk about [schnorr signatures][topic
+  schnorr signatures] as an upgrade on Bitcoin's original ECDSA
+  signatures because they make it easier to implement various
+  cryptographic tricks, but the schnorr signature algorithm predates the
+  DSA algorithm that ECDSA is based upon.  Indeed, DSA was created in
+  part to circumvent the [patent on schnorr signatures][schnorr patent]
+  and it was [reported][gov report] that "there were allegations of
+  infringement from Mr. Claus Schnorr and Public Key Partners (PKP)"
+  that DSA violated Schnorr's patent.  That patent expired in 2008.
+
+- **Un-Schnorr what name to use:** although it didn't work out, there
+  was a [suggestion][dryja bn sigs] early in the development of adapting
+  schnorr signatures for Bitcoin that Claus Schnorr's name shouldn't be
+  used in association with them because his patent on them prevented the
+  widespread use of a valuable cryptographic technique for over 20
+  years.
```

Comment:
more a comment, no need to change anything: I don't know him personally but in his defend, those were just different times. I think nowadays the idea that patents are not a good idea is much more wide-spread in the applied crypto community. Even Phil Rogaway who wanted to use them for good (no military use etc.) apparently has moved away from the idea https://mailarchive.ietf.org/arch/msg/cfrg/qLTveWOdTJcLn4HP3ev-vrj05Vg/

Reply:
Schnorr wasn't trying to "do good", he was just trying to make money. Though there isn't anything wrong with making moneyit's not like the idea wouldn't have existed for a decade without him (very similar constructions existed previously-- like the ECDSA-ish ElGamal signature scheme).

Schnorr's unwillingness to open up the patent was controversial even back in the 90s.  Schnorr actually went around [claiming anyone using any kind of discrete log signature violated his patent](https://web.archive.org/web/19991117143502/http://grouper.ieee.org/groups/1363/letters/SchnorrMar98.html) (but due to the aforementioned ElG scheme his threats against DSA weren't credible and only stopped a smaller number of parties).

Reply:
Sure he was trying to make money and indeed, there's nothing wrong with. My thought was just that people nowadays understand that patenting your new crypto algorithm is the best way to ensure you won't make any money with it...